apps/server/src/tasks/sendTestConfirmationEmail.ts

```diff
@@ -0,0 +1,20 @@
+import { fixedGroups } from '@raise/shared';
```

Comment:
Was this supposed to be in #310 instead?

Reply:
will fix this in the second commit